pkg/cmd/cli/cmd/process.go

```diff
@@ -122,7 +124,10 @@ func RunProcess(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []
 	// When storedTemplate is not empty, then we fetch the template from the
 	// server, otherwise we require to set the `-f` parameter.
 	if len(storedTemplate) > 0 {
-		templateObj, err := client.Templates(namespace).Get(storedTemplate)
+		if parts := strings.Split(storedTemplate, "/"); len(parts) > 1 {
```

Comment:
Actually, no it can't.  Hrm...  I guess this is fine.  But you need to validate parts == 1 or 2, not > 1